docker/README.md

```diff
@@ -66,14 +69,41 @@ Explanation of parts:
 * `-p 4190:8080` maps the internal algod REST API to local port 4190
 * `-e NETWORK=` can be set to any of the supported public networks.
 * `-e FAST_CATCHUP=` causes fast catchup to start shortly after launching the network.
-* `-e TELEMETRY_NAME=` enables telemetry reporting to Algorand for network health analysis.
+* `-e TELEMETRY_NAME=` enables telemetry reporting to Algorand for network health analysis. The value of this variable takes precedence over the GUID in `/etc/logging.config`.
 * `-e TOKEN=` sets the REST API token to use.
 * `-v ${PWD}/data:/algod/data/` mounts a local volume to the data directory, which can be used to restart and upgrade the deployment.
 
 ## Mounting the Data Directory
 
 The data directory located at `/algod/data`. Mounting a volume at that location will allow you to shutdown and resume the node.
 
+### Handling Permission Errors
+
+The container executes in the context of the `algorand` user with it's own UID and GID. If experiencing permission errors when mounting a volume, try one of the following:
+
+1. Using a named volume
+
+    ```bash
+    docker run -it --rm -d -v algod-data:/algod/data algorand/algod
+    ```
+
+2. Specifying UID/GID of the container
+
+    ```bash
+    docker run -it --rm -d -v /srv/data:/algod/data -u $UID:$GID algorand/algod
+    ```
+
+3. Relabeling the contents of the volume
+
+    ```bash
+    docker run -it --rm -d -v /srv/data:/algod/data:Z algorand/algod
+    ```
```

Comment:
Not strictly necessary now, but each one of these options would benefit from being fleshed out.

For example I'm guessing "Using a named volume" has a separate command to create a docker-managed volume?

The "relabeling" option looks very strange. The first two generally make some logical sense about why they would solve the problem, but that one is a mystery to me.